modules/packages/LinearAlgebra.chpl

```diff
@@ -2418,16 +2421,16 @@ proc kron(A: [?ADom] ?eltType, B: [?BDom] eltType) {
 }
 
 /*
-  Matrix exponential using Pade approximation. This method returns N*N matrix which
-  is Matrix exponential of `A`
+  Matrix exponential using Pade approximation. This method returns square matrix which
+  is matrix exponential of ``A``.
 
-  :arg A: Expects an N*N square matrix.
+  :arg A: Expects a square matrix.
   :type A: `A`
 
-  :arg useExactOneNorm: boolean value specifying if the onenorm has to be exact. Defaults to true.
+  :arg useExactOneNorm: boolean value specifying if the onenorm has to be exact. Defaults to `true`.
   :type useExactOneNorm: bool
 
-  :throws LinearAlgebraError: If Input Matrix is not Square Matrix.
+  :throws LinearAlgebraError: If input matrix is not square matrix.
```

Comment:
This is also missing articles, e.g. "If _the_ input matrix is not _a_ square matrix"